crowd.properties

```diff
@@ -1,25 +1,19 @@
 category=Integration
 description=Provide the ability to authenticate again Crowd
-homepageUrl=https://github.com/SonarCommunity/sonar-crowd
+homepageUrl=https://github.com/deepy/sonar-crowd
 devVersion=
-publicVersions=2.1,2.1.1
-archivedVersions=1.0,2.0
+publicVersions=2.1.2
+archivedVersions=1.0,2.0,2.1,2.1.1
```

Comment:
2.1 offers support for 7.0-7.2. Did you mean to remove the ability to download it through the Marketplace?

Reply:
Yeah, 2.1 and 2.1.1 are functionally identical but 2.1.1 uses less deprecated functionality so it seemed logical to remove 2.1